karapace/kafka_rest_apis/__init__.py

```diff
@@ -464,6 +464,15 @@ async def validate_schema_info(self, data: dict, prefix: str, content_type: str,
                 content_type=content_type,
                 status=HTTPStatus.BAD_REQUEST,
             )
+        except SchemaRetrievalError:
+            self.r(
+                body={
+                    "error_code": RESTErrorCodes.SCHEMA_RETRIEVAL_ERROR.value,
+                    "message": f"Error when registering schema. format = {schema_type}, subject = {topic}-{prefix}",
+                },
+                content_type=content_type,
+                status=HTTPStatus.REQUEST_TIMEOUT,
```

Comment:
Why 408 timeout? Based on the PR description better alternatives would be 406 not acceptable, 409 conflict,  422 unprocessable entity.

Reply:
408 is same what CSR implementation returns, so I reused it. I admit it is not the best suitable though.

Reply:
Then it is the correct choice.